source/documentation/fuseki2/fuseki-logging.md

```diff
@@ -53,6 +53,16 @@ which defaults to `/etc/fuseki` on Linux.
 For the standalone webapp server, `FUSEKI_BASE` defaults to directory `run/`
 within the directory where the server is run.
 
+The property `fuseki.loglogging` can also be set to `true` for additional logging.
```

Comment:
Would it be worth mentioning the fuseki environment variable `FUSEKI_LOGLOGGING also`, .ie.

The property `fuseki.loglogging` or the environment variable `FUSEKI_LOGLOGGING` can also be set to `true` for additional logging? 

I see it used in https://github.com/apache/jena/blob/main/jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/system/FusekiLogging.java#L68

Reply:
@OyvindLGjesdal -- would you like to put in a PR?

The "meta" logging isn't the focus of this PR.

Merely FYI:
There is a companion setup in  `LogCtl`: `JENA_LOGLOGGING` `jena.loglogging`.

But Fuseki logging is more exposed to the runtime environment and a bit more complicated especially in war file form.